hwdb.d/60-keyboard.hwdb

```diff
@@ -1089,7 +1089,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnIBM*:pn18666TU:*
 # IdeaPad
 evdev:name:Ideapad extra buttons:dmi:bvn*:bvr*:bd*:svnLENOVO*:pn*:*
  KEYBOARD_KEY_0d=rfkill                                 # airplane mode switch (toggle all wireless devices)
- KEYBOARD_KEY_08=f20                                    # micmute
+ KEYBOARD_KEY_08=micmute
```

Comment:
AFAICT this has been the default kernel mapping, so this can be dropped instead of changing it.